runtime/commonMain/src/kotlinx/serialization/modules/PolymorphicModuleBuilder.kt

```diff
@@ -51,7 +51,7 @@ public class PolymorphicModuleBuilder<Base : Any> internal constructor(
      * @see addSubclass
      */
     @ImplicitReflectionSerializer
-    public inline fun <reified T : Base> addSubclass(): Unit = addSubclass(T::class, T::class.serializer())
+    public inline fun <reified T : Base> subclass(): Unit = addSubclass(T::class, serializer())
```

Comment:
It's likely someone already used this function, so we'd better provide deprecation alias.

In other parts, PR is ok (modulo function removing we've already discussed)